Formula/libpaper.rb

```diff
@@ -0,0 +1,31 @@
+class Libpaper < Formula
+  desc "Library for handling paper characteristics"
+  homepage "https://github.com/rrthomas/libpaper"
+  url "https://github.com/rrthomas/libpaper/releases/download/v2.0.10/libpaper-2.0.10.tar.gz"
+  sha256 "34938c376ad9ba706dd0f1b30e3de0cb5fe54b47e528ae8a7f1fee062dc42c72"
+  license "LGPL-2.1-or-later"
+
+  depends_on "help2man" => :build
+
+  def install
+    system "./configure", *std_configure_args
```

Comment:
```suggestion
    system "./configure", *std_configure_args, "--synsconfdir=#{etc}"
```
Let's not install user configuration files into the cellar.